ADMIN.adoc

```diff
@@ -0,0 +1,36 @@
+= Repository administrator responsibilities
+
+== Administrator rotation schedule
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be an administrator is voluntary.
+To volunteer to be an administrator, contact @cbudz or @bergerhoffer.
+
```

Comment:
@inoxx03 please insert at the end of this section that each administration rotation is marked on the Style Guide shared calendar, and admins will receive a notification and calendar invite when it's their turn.